tests_e2e/tests/scripts/agent_persist_firewall-verify_firewall_rules_on_boot.py

```diff
@@ -70,17 +70,21 @@ def cron_log_checks():
         if not os.path.exists(cron_log):
             raise Exception("Cron log file not found: {0}".format(cron_log))
         with open(cron_log) as f:
-            cron_logs_lines = list(map(lambda _: _.strip(), f.readlines()))
-        if not cron_logs_lines:
+            cron_log_content = f.read()
```

Comment:
including the log of the cron job in the log of the test makes debugging a lot easier/faster